### DISCUSSION TITLE
multi-accounts

### DISCUSSION THREAD

**Discussion by GuShaocheng** [TriliumNext/Trilium#3024] [Q&A]
Once you initialised the server, are you able to make a new account?

**Comment by sigaloid** [comment]
Here's some info on multi accounts from the wiki: https://github.com/zadam/trilium/wiki/FAQ#multi-user-support

**Reply by ivityc** [reply]
I understand that we cannot have multiple account working on the same set of notes.
But would it be possible to have one single server / database where 2 different users works on their personal notes?
no co-editing/sharing/interaction required. 
Each user would see only his personal dB as single user.
Just the server resources are shared amongst multiple user, to avoid to create different servers with allocated memory/cpu/disk etc.
Thanks

**Reply by sottey** [reply]
You would need two separate servers. The server, on startup, is told where the data is stored. 

Having two docker containers running on different ports would not take a lot of resources.

**Reply by zadam** [reply]
What you describe sounds similar to "multi-tenancy", but this is not supported. You will need to run two node.js processes (in some way, e.g. as two docker containers).
